app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,34 @@
+class VotesController < ApplicationController
+  before_action :get_vote, only: [:show, :edit, :update, :destroy]
+
+  def new
+    @vote = Vote.new
+  end 
+
+  def create
+    @vote = Vote.create(vote_params)
+  end
+
+  def edit;end
```

Comment:
@GerryLarios 
Would it be alright to take this line out to be consistent with the show action?

Reply:
Sure thing! let me remove it.